File: app/src/main/java/per/goweii/wanandroid/event/MessageDeleteEvent.java
package per.goweii.wanandroid.event;

import per.goweii.wanandroid.module.mine.model.MessageBean;

/**
 * @author CuiZhen
 * @date 2019/5/17
 * GitHub: https://github.com/goweii
 */
public class MessageDeleteEvent extends BaseEvent {

    private MessageBean bean;

    private MessageDeleteEvent(MessageBean bean) {
        this.bean = bean;
    }

    public static void post(MessageBean bean) {
        new MessageDeleteEvent(bean).post();
    }

    public MessageBean getMessageBean() {
        return bean;
    }
}

File: app/src/main/java/per/goweii/wanandroid/event/MessageDeleteEvent.kt
package per.goweii.wanandroid.event

import per.goweii.wanandroid.module.mine.model.MessageBean

class MessageDeleteEvent private constructor(val messageBean: MessageBean) : BaseEvent() {
    companion object {
        @JvmStatic
        fun post(messageBean: MessageBean) = MessageDeleteEvent(messageBean).post()
    }
}